libs/dh/metering-point/feature-search/src/lib/form/dh-metering-point-search-form.component.spec.ts

```diff
@@ -0,0 +1,215 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ActivatedRoute } from '@angular/router';
+import { en as enTranslations } from '@energinet-datahub/dh/globalization/assets-localization';
+import { getTranslocoTestingModule } from '@energinet-datahub/dh/shared/test-util-i18n';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { render, screen, fireEvent, waitFor } from '@testing-library/angular';
+import { TestBed } from '@angular/core/testing';
+import userEvent from '@testing-library/user-event';
+
+import {
+  validMeteringPointId,
+  invalidMeteringPointId,
+} from '@energinet-datahub/dh/shared/test-util-metering-point';
+
+import { DhMeteringPointSearchComponent } from '../dh-metering-point-search.component';
+import {
+  DhMeteringPointSearchFormComponent,
+  DhMeteringPointSearchFormScam,
+} from './dh-metering-point-search-form.component';
+
+describe(DhMeteringPointSearchFormComponent.name, () => {
+  async function setup() {
+    const { fixture, navigate } = await render(
+      DhMeteringPointSearchFormComponent,
+      {
+        imports: [
+          NoopAnimationsModule,
+          getTranslocoTestingModule(),
+          DhMeteringPointSearchFormScam,
+        ],
+        routes: [
+          {
+            path: `metering-point/search`,
```

Comment:
suggestion: use the existing variables to construct the path. Also later in the file.